netbox/utilities/models.py

```diff
@@ -33,9 +32,37 @@ def to_objectchange(self, action):
         Return a new ObjectChange representing a change made to this object. This will typically be called automatically
         by extras.middleware.ChangeLoggingMiddleware.
         """
+        from extras.models import ObjectChange
         return ObjectChange(
             changed_object=self,
             object_repr=str(self),
             action=action,
             object_data=serialize_object(self)
         )
+
+
+class CommentModel(models.Model):
```

Comment:
Using abstract models to add individual fields is overkill. At most we would implement a custom field class, and even that seems excessive. All we need to do is standardize and document the field attributes.

Reply:
Perhaps just go with a simple constant for length, that looks to be all that we need to standardize on.